css/css-grid/css-grid_image-gallery/css/styles.css

```diff
@@ -52,6 +54,7 @@ body {
 .teaser-grid__teaser--winter {
   background-image: linear-gradient(rgba(0, 0, 0, 0.4), rgba(0, 0, 0, 0.4)),
     url("https://source.unsplash.com/random/600x300?winter");
+  grid-area: winter;
```

Comment:
As it's said in the task to write only 2 lines of code, so this line is not needed.